src/google_cloud_debugger/google_cloud_debugger_lib/dbg_breakpoint.cc

```diff
@@ -286,13 +287,11 @@ HRESULT DbgBreakpoint::PopulateExpression(
   }
 
   if (bfs_queue.size() != 0) {
-    int half_available_breakpoint_size =
-        (kMaximumBreakpointSize - breakpoint->ByteSize()) / 2;
     return VariableWrapper::PerformBFS(
         &bfs_queue,
-        [breakpoint, half_available_breakpoint_size]() {
-          return breakpoint->ByteSize() > half_available_breakpoint_size;
-        },
+        // No terminating condition so the BFS will
+        // only end when all items in the queue is processed.
```

Comment:
I think we need to still have the basic check to not go over 'kMaximumBreakpointSize' but beyond that I think we can allow everyting.